consensus/tendermint/tendermint.go

```diff
@@ -294,17 +301,21 @@ func (t *Tendermint[V, H, A]) startRound(r round) Action[V, H, A] {
 }
 
 type timeout struct {
-	s step
-	h height
-	r round
+	S step   `cbor:"s"`
+	H height `cbor:"h"`
+	R round  `cbor:"r"`
```

Comment:
```suggestion
	Step step `cbor:"s"`
	Height height `cbor:"h"`
	Round round  `cbor:"r"`
```
to reduce conflict with #2786